vst_patch.patch

```diff
@@ -0,0 +1,39 @@
+diff --git a/chuck/include/chuck_oo.h b/chuck/include/chuck_oo.h
+index 0811988..08f777f 100644
+--- a/chuck/include/chuck_oo.h
++++ b/chuck/include/chuck_oo.h
+@@ -80,11 +80,11 @@ public:
+ 
+ public:
+     // add reference (ge: april 2013: made these virtual)
+-    virtual void add_ref();
++    virtual void add_ref() {};
```

Comment:
Could you submit a pull request against https://github.com/ccrma/chuck so that this patch isn't necessary here?

Reply:
Yes but I'm reluctant to do that because I don't know if it would have unwanted side effects.